internal/runner/runner.go

```diff
@@ -72,6 +73,59 @@ func (d *Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+// Multi will both print the output for the user and return it to callers.
+// Useful for debugging, e.g. if the importer calls terraform import but it freezes without returning output.
+// Inspired by https://blog.kowalczyk.info/article/wOYk/advanced-command-execution-in-go-with-osexec.html
+type Multi struct {
+	Quiet bool
+}
+
+// CmdRun executes the command aand prints stdout and stderr without returning either.
+func (d *Multi) CmdRun(cmd *exec.Cmd) error {
+	if !d.Quiet {
+		log.Printf("Running: %v", cmd.Args)
+	}
+
+	var stderr bytes.Buffer
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = io.MultiWriter(os.Stderr, &stderr)
```

Comment:
why do we need this? doesn't the output stream to stderr anyway so it is visible to the user? this will cause the stderr to be printed twice (once through stderr then again through the error msg)

Reply:
You're right. I'll change it.